scripts/sync-to-config-folder.sh

```diff
@@ -180,4 +179,4 @@ git push -u origin "${FORK_RELEASE_BRANCH}":$RELEASE_BRANCH
 gh pr create --title "🥳 ${BINARY_BASE} ${VERSION} Automated manifest sync! 🥑" \
     --body "${PR_BODY}" --repo ${REPO} --base ${RELEASE_BRANCH}
 
-echo "✅ Manifest folder PR created for $RELEASE_BRANCH"
+echo "✅ Manifest folder PR created for{RELEASE_BRANCH}"
```

Comment:
Should there be a `$` with {RELEASE_BRANCH}` ?

Reply:
Ah yes this was a typo, thank you for catching!